frontend/hub/collections/CollectionPage/CollectionImportLog.tsx

```diff
@@ -27,67 +30,195 @@ export function CollectionImportLog() {
+              >
+                <pre>{collectionImport?.error?.traceback}</pre>
+              </Alert>
+            )}
+            <div>
+              <CodeBlock
                 style={{
-                  color: getPatternflyColor(
-                    message.level === 'INFO'
-                      ? PFColorE.Default
-                      : message.level === 'WARNING'
-                      ? PFColorE.Warning
-                      : message.level === 'ERROR'
-                      ? PFColorE.Danger
-                      : PFColorE.Disabled
-                  ),
+                  backgroundColor: 'var(--pf-v5-global--palette--black-850)',
+                  position: 'relative',
                 }}
               >
-                {message.message}
-              </div>
-            ))}
-          </CodeBlock>
-        </Stack>
-      </PageSection>
+                <NavigationArrow
+                  direction="down"
+                  onClick={() => ref.current?.scrollIntoView({ block: 'end', behavior: 'smooth' })}
+                />
+
+                <NavigationArrow
+                  direction="up"
+                  onClick={() =>
+                    ref.current?.scrollIntoView({ block: 'start', behavior: 'smooth' })
+                  }
+                />
+
+                {collectionImport?.messages?.map((message, index) => (
+                  <div
+                    key={index}
+                    style={{
+                      color: getColor(message.level),
+                    }}
+                  >
+                    {message.message}
+                  </div>
+                ))}
+                <br />
+                <div
+                  key={'done'}
+                  style={{
+                    color: PFColorE.Green,
+                  }}
+                >{t`Done`}</div>
+              </CodeBlock>
+            </div>
+          </Stack>
+        </PageSection>
+      </div>
     </Scrollable>
   );
 }
+
+function getColor(messageLevel: string) {
+  const res = getPatternflyColor(
+    messageLevel === 'WARNING'
+      ? PFColorE.Warning
+      : messageLevel === 'ERROR'
+      ? PFColorE.Danger
+      : PFColorE.Disabled
+  );
+
+  if (messageLevel === 'INFO') {
+    return 'white';
+  }
+  return res;
+}
+
+const arrowStyle = `
+  color: white;
+  &:hover {
+    cursor: pointer;
+  }
+  position: absolute;
+  margin-bottom: 10px;
+  margin-right: 10px;
+  font-size: 120%;
+`;
+
+const StyledArrowDown = styled.div`
+  ${arrowStyle}
+  right: 0; /* Aligns the icon to the right */
+  top: 0; /* Aligns the icon to the top */
+`;
+
+const StyledArrowUp = styled.div`
+  ${arrowStyle}
+  right: 0;
+  bottom: 0;
+`;
+
+function NavigationArrow(props: { direction: 'up' | 'down'; onClick: () => void }) {
+  const className = `fa fa-arrow-circle-${props.direction} clickable`;
+
+  const Component = props.direction === 'up' ? StyledArrowUp : StyledArrowDown;
+
+  return (
+    <Component>
+      <span
```

Comment:
Could we use a native button here to avoid any accessibility issues? Did I misread the intent here?

Reply:
Is it necessary? This is from old code, but changing this to button broke the css of element (it looks badly), which will need some effort to repair it.

Reply:
I will merge it for now and we can consult it after you return from PTO, I dont want this PR to be blocked for too long.